### DISCUSSION TITLE
blankpage after login on URL

### DISCUSSION THREAD

**Discussion by fadhil93** [Leantime/leantime#1259] [General]
Hi
I installing leantime on VM nutanix.
DNS using cloudflare
SSL engine using certbot
My php7.4
ubuntu 20.04
my apache.conf

<VirtualHost *:80>
        DocumentRoot /var/www/html/leantime/public/

<Directory /var/www/html/leantime/public>
                AllowOverride AuthConfig FileInfo Indexes Limit
                Require all granted
</Directory>

        ErrorLog ${APACHE_LOG_DIR}/error.log
        CustomLog ${APACHE_LOG_DIR}/access.log combined

</VirtualHost>

<VirtualHost *:443>

        DocumentRoot /var/www/html/leantime/public/

        ErrorLog ${APACHE_LOG_DIR}/error.log
        CustomLog ${APACHE_LOG_DIR}/access.log combined

        SSLEngine on
        SSLCertificateFile /etc/letsencrypt/live/**********/cert.pem
        SSLCertificateKeyFile /etc/letsencrypt/live/*************/privkey.pem

</VirtualHost>

my problem is 
I can login into leantime using local ip. 

![image](https://user-images.githubusercontent.com/62331653/210707963-65f9eaf9-f1bc-411d-a0f9-1dffb9d20809.png)

but i can't login into leantime on my URL it will turn white page after I login

**Comment by marcelfolaron** [comment]
Hi, I know this is over a month old, but from the screenshot it seems you are using an old version. Please updated to the latest version.
